SparkPost/2web2ui
# SA-2062 - (UI) Redesign seedlist page

[SA-2062](https://sparkpost.atlassian.net/browse/SA-2062)

### What Changed

- Redesigns Seed List page to match the mocks attached to the ticket
- Uses `useSparkPostQuery` in favor of Redux
- Removes unused Redux code
- Migrates a selector from `src/selectors` to `src/helpers`
- Adds supporting Cypress and Jest tests

### How To Test

- Visit `/inbox-placement/seedlist` and verify the page meets the criteria outlined in the ticket
- Verify the seed list is requested successfully and is copyable using the primary button
- Verify a CSV of the seed list can be downloaded using the secondary button
- Verify tests accurately validate the acceptance criteria

### To Do
